modules/nw-ingress-integrating-route-secret-certificate.adoc

```diff
@@ -0,0 +1,62 @@
+//
+// * ingress/routes.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-ingress-integrating-route-secret-certificate_{context}"]
+= Integrating Third-Party Certificate Management
+
+The following workflow describes how to integrate third-party certificate management solutions, such as {cert-manager}, using the certificate reference field outlined under API Extensions.
+
+A new `.spec.tls.externalCertificate` field has been added to the Route's `.spec.tls` configuration. This field can be used to specify the name of a secret containing the certificate data, replacing the need to use `.spec.tls.certificate` and .`spec.tls.key`.
+
+.Prerequisites
```

Comment:
We should also add the feature-gate and `custom-host` permission related informations here

Reply:
For enabling `TechPreviewNoUpgrade` feature set please refer to https://docs.openshift.com/container-platform/4.8/nodes/clusters/nodes-cluster-enabling-features.html#nodes-cluster-enabling-features-cli_nodes-cluster-enabling

Reply:
I will add the Tech Preview feature snippet.